modules/aws/s3.go

```diff
@@ -115,6 +116,35 @@ func CreateS3BucketE(t *testing.T, region string, name string) error {
 	return err
 }
 
+// PutS3BucketVersioningE creates an S3 bucket versioning configuration in the given region against the given bucket name, WITHOUT requiring MFA to remove versioning.
+func PutS3BucketVersioning(t *testing.T, region string, bucketName string) {
+	err := PutS3BucketVersioningE(t, region, bucketName)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I know this file doesn't have this pattern yet, but we've started shortening this to a one liner: `require.NoError(t, err)`. Can you apply to these functions?

Reply:
Implemented